src/OrchardCore.Build/OrchardCore.Commons.props

```diff
@@ -23,6 +23,7 @@
     <IncludeSourceRevisionInInformationalVersion>true</IncludeSourceRevisionInInformationalVersion>
     <ContinuousIntegrationBuild Condition="'$(CI)' == 'true'">true</ContinuousIntegrationBuild>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageIcon>icon.png</PackageIcon>
```

Comment:
The docs recommend 128x128px images, not 220px ones: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-an-icon-image-file

Reply:
I guess higher resolution won’t harm

Reply:
I wouldn't guess ;). Perhaps not, but I have just a vague idea why they recommend that resolution, and without knowing better I wouldn't go against it.

Reply:
So people like me don’t put lower resolution, as I did in prev commit